src/core/propertyTypes.js

```diff
@@ -117,6 +118,10 @@ function srcParse (value) {
   var el = selectorParse(value);
   if (el) { return el.getAttribute('src'); }
 
+  if (value.charAt(0) !== '#') {
```

Comment:
Right; I was following your logic here:  https://github.com/aframevr/aframe/issues/899

I'd assumed you made ID selection a standard.  Is your `#` comment no longer true?